goalc/compiler/Compiler.cpp

```diff
@@ -21,11 +21,12 @@ using namespace goos;
 Compiler::Compiler(GameVersion version,
                    const std::optional<REPL::Config> repl_config,
                    const std::string& user_profile,
-                   std::unique_ptr<REPL::Wrapper> repl)
+                   std::unique_ptr<REPL::Wrapper> repl,
+                   const std::string& iso_path)
     : m_version(version),
       m_goos(user_profile),
       m_debugger(&m_listener, &m_goos.reader, version),
-      m_make(repl_config, user_profile),
+      m_make(repl_config, user_profile, iso_path),
```

Comment:
rather than adding a new argument, this looks like something that could be set on the `repl_config`, it's something that is already passed along to the `m_make` instantiation as well.

Reply:
I wrote the latest commit to handle this.